prog/kube-peers/annotations.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/pkg/errors"
+
+	v1 "k8s.io/api/core/v1"
+	kubeErrors "k8s.io/apimachinery/pkg/api/errors"
+	api "k8s.io/apimachinery/pkg/apis/meta/v1"
+	wait "k8s.io/apimachinery/pkg/util/wait"
+	kubernetes "k8s.io/client-go/kubernetes"
+)
+
+type configMapAnnotations struct {
+	Name      string
```

Comment:
Perhaps a short clarification would be good here: is it a name of the annotation or a name of the ConfigMap?